### ISSUE TITLE
Named parameter is undefined with && parameters

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#803]
On behalf of SAV0

This declaration triggers a false positive on rule "Undefined named include":
```
{ myinc.i &param1="xxx" &&param2="yyy" }
```

Valid syntax, but requires a reference to `{&&param2}`, which is a code smell to my mind. And incidentally not the case in this code, so the second parameter was completely ignored.
Same rule could be reused with a different message for this case, or that can be added as a new rule (e.g. Invalid name for include parameter)

**Comment by gquerret** [comment]
Fixed in develop
